appveyor.yml

```diff
@@ -1,10 +1,17 @@
-version: 1.0.{build}-Alpha
+version: 1.8.1.{build}
 os: Visual Studio 2015
 
 cache:
   - '%LocalAppData%\NuGet\Cache'
   - '%Temp%\nethook2-dependencies -> Resources\NetHook2\SetupDependencies.ps1'
 
+assembly_info:
+  patch: true
+  file: AssemblyInfo.cs
+  assembly_version: "1.8.1"
```

Comment:
If my understanding is correct and this patches our AssemblyInfo.cs, could we write a comment there that it's automatically handled by CI for deployments.